### DISCUSSION TITLE
Latest Sprint Plan (Feb 28 to Mar 18)

### DISCUSSION THREAD

**Discussion by ananya-jain1** [Sunbird-cQube/community#762] [Announcements]
Hey folks,
[Here](https://project-sunbird.atlassian.net/jira/software/c/projects/CQ/boards/117/backlog?issueLimit=100) is the latest sprint plan from Feb 28 to Mar 18. 

At the end of this sprint, the adopter will be able to install cQube or upgrade from v4.1 Beta to v5.0 seamlessly, ingest spec-compliant data for 12 out-of-the-box programs, & leverage RBAC-enabled desktop-based visualisation layer of cQube for all 12 programs. The adopter will also be able to add a new program or indicator via APIs.

@surendrasinghs @sukhpreetsamagra @rahul101001000 @ChakshuGautam @sagarihanda

**Comment by ananya-jain1** [comment]
@SanthoshVasabhaktula

**Comment by surendrasinghs** [comment]
@ananya-jain1 the sprint plan mentions "New Approach". Can you please explain and share details?
